tests/DYCOMS2D/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_test(calc calc ${CMAKE_BINARY_DIR})
 find_package(HDF5 COMPONENTS CXX REQUIRED QUIET)
 find_package(Boost COMPONENTS iostreams system filesystem REQUIRED)
 
-foreach(micro lgrngn)
+foreach(micro lgrngn blk_1m blk_2m)
```

Comment:
Did you run the dycoms2d test? It is disabled by default.

Reply:
I did not run the dycoms2d test. I wasn't sure if since it was disabled, that you plan to remove it in the future. But I did run dycoms_rf02 in 2D with bulk_2m microphysics. Those results are in the plot I sent earlier.

Reply:
I did not run the dycoms2d test. I wasn't sure if since it was disabled, that you plan to remove it in the future. But I did run dycoms_rf02 in 2D with bulk_2m microphysics. Those results are in the plot I sent earlier.

Reply:
Then I think it's better to revert changes to the dycoms2d test files: calc, gnuplot, hdf, plot_lgrngn. 
Please keep plot_blk_1m and plot_blk_2m.

Reply:
Reverted those files to current master.